app/views/provider_mailer/volunteer_response_email.erb

```diff
@@ -0,0 +1,31 @@
+<h1 style="margin-top: 0; color: #333333; font-size: 22px; font-weight: bold; text-align: left;" align="left"><%= @provider.first_name %>, a volunteer named <%= @volunteer.full_name %> has responded to your help request!</h1>
+
+<p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;">
+  <%= @volunteer.first_name %> has offered to help with
+  <%= @response.requests.map { |request| Provider::REQUEST_TYPES[request].downcase }.to_sentence %>.
+</p>
+
+<p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;">
+  They are available <%= @response.availabilities.to_sentence %>.
+
+  <% if @response.description.present? %>
+    Here is what they wrote:
+    <p>
+      <%= @response.description %>
+    </p>
+  <% end %>
+</p>
+
+<% if @response.phone.present? %>
+  <p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;">
+    Phone: <%= @response.phone %>
+  </p>
+<% end %>
+
+<% if @response.social.present? %>
+  <p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;">
+    Social account: <%= @response.social %>
+  </p>
+<% end %>
+
+<p style="font-size: 16px; line-height: 1.625; color: #51545E; margin: .4em 0 1.1875em;">You can reply to this email (or send them a note at <a href="mailto:<%= @volunteer.email %>"><%= @volunteer.email %></a>) to start a conversation with <%= @volunteer.first_name %> about your needs. When your request is met, please update the request using the link that we originally sent you.</p>
```

Comment:
Minor: can we change "please update the request using the link that we originally sent you." to "please update the request by clicking [here](url)" or something similar. 

I'm thinking of the case where a provider gets a response and realizes they no longer want responses. Their probably more likely to update their request if they don't have to go find the link in their email.

Reply:
So, I was thinking about that, but since people are going to reply to these emails to contact the volunteers, I didn't want them to accidentally send along their edit links with tokens.

Reply:
Open to ideas about how to deal with this.

Reply:
Great point! Probably not worth the time in this PR but the budget software I use sends one-time use tokens  for logging in. There might be something like that we could do eventually.